### ISSUE TITLE
telegram notifications skip every 10 minutes feature

### ISSUE THREAD

**Issue by xjustbmw** [VorlonCD/bi-aidetection#320]
Is it possible to receive only one positive notification every 10 minutes (for all cameras).
The rest should be skipped.
I have several cameras setup and when Im outside the phone beeps all the time while I move :)
Also made a tasker profile to silence the phone , but the battery drains like crazy from these photos coming in:)

Tried playing with cooldown time, but the photos are just delayed in this way.

**Comment by xjustbmw** [comment]
instant reply :)
many thanks for your help
i was using an old version of aitool indeed :) october 2020 (last release)
with latest deepstack im getting error when trying to start, yet the deepstack in debug mode seems to start , but http://127.0.0.1:50000/v1/vision/detection seems to load a 404 page
is it a different url now ?

**Comment by xjustbmw** [comment]
all good
had to manually kill server.exe

**Comment by VorlonCD** [comment]
Does that still happen with latest AITOOL version?   It supports the latest 'Windows' deepstack release 2022.01.01.

**Comment by VorlonCD** [comment]
Sends back to AITOOL.  When working correctly upon startup AITOOL will send you a list of supported commands such as REBOOT, SHUTDOWN, PAUSE, etc.

**Comment by xjustbmw** [comment]
just got the message back that all cameras are paused for x minutes :)
this is the only way to stop it sending messages ?

**Comment by VorlonCD** [comment]
No, cooldown should be working of course, not sure why you were having trouble.  Try 600 or 1200 seconds.  Also, you can simply go to Actions, and  uncheck "Send alert images to telegram" if you dont want it to send images at all.   And "Dynamic masking" will stop sending alerts if it keeps seeing an object in the same place, like a car in your driveway.

**Comment by VorlonCD** [comment]
Oh, maybe you want to use the per-camera COOLDOWN setting instead of the telegram specific one?   Cameras tab > Action settings > Cooldown Seconds

**Comment by xjustbmw** [comment]
auch :) I will try that as well. In the last stable version i think it behaved the same.

but, I have about 12 cameras pointing at the same garden :) that will trigger 12 times untill all of them ... cooldown :)

**Comment by VorlonCD** [comment]
Hmm try turning off the QUEUED checkbox for each camera.   That could be related since queued actions behave a little different.   If you hover over the checkbox you will see it says "_it is possible this will affect how the telegram cooldown feature works_"

**Comment by xjustbmw** [comment]
indeed. queue was on for all cameras
you think this also affects the main telegram cooldown setting ?

**Comment by VorlonCD** [comment]
yes it does affect that.  When queued it simply waits the cooldown time but doesnt actually skip it - just like you are seeing.
What I really need to set up are camera GROUPS so multiple cameras can have the same cooldown timing.

**Comment by xjustbmw** [comment]
Yes. that would be amazing. :)
Also groups can handle same object detection : example cars in one, persons in others :)

I remember we spoke about 1 year ago about the object minimum size (I had in cats falsely detected as persons by deepstack, and was able to filter them out by their size after you added the feature)


Will test tomorrow the cooldown without queue. quite sure it will work though :)

**Comment by xjustbmw** [comment]
little updated as promised. all is working perfectly:)
